WonderMake/Program/Program.h

```diff
@@ -30,22 +30,19 @@ class Program
 	void SetupCallbacks();
 	void OnWindowSizeChanged([[maybe_unused]]GLFWwindow* Window, i32 X, i32 Y);
 
-	//-------KEEP THIS IN ORDER-------
 	DockSpace myDockSpace;
 	ConsoleLogger myLogger;
 	FileLogger myFileLogger;
 	ImGuiLogger myImGuiLogger;
-	SystemPtr<Window> myWindowPtr;
-	SystemPtr<Renderer> myRendererPtr;
-	SystemPtr<EngineUniformBuffer> myEngineUniformBufferPtr;
-	//-------KEEP THIS IN ORDER-------
 
-	SystemPtr<Camera> myCameraPtr;
-	SystemPtr<TimeKeeper> myTimeKeeperPtr;
 	SystemPtr<InputSystem> myInputSystem;
-	SystemPtr<DebugSettingsSystem> myDebugSettingsPtr;
 	SystemPtr<AudioManager> myAudioManagerPtr;
 	SystemPtr<FileWatcher> myFileWatcher;
+	SystemPtr<TimeKeeper> myTimeKeeperPtr;
+	SystemPtr<EngineUniformBuffer> myEngineUniformBufferPtr;
+	SystemPtr<Window> myWindowPtr;
+	SystemPtr<Camera> myCameraPtr;
+	SystemPtr<Renderer> myRendererPtr;
```

Comment:
Assuming these will disappear later on?

Reply:
Correct. To remove them I'd need to make some refactoring which is beside what this pull request is intended to do.